subprojects/gradle/instrumentation-tests/src/test/kotlin/com/avito/instrumentation/report/WriteJUnitReportActionTest.kt

```diff
@@ -93,16 +93,16 @@ internal class WriteJUnitReportActionTest {
     }
 
     private fun mockData(testRunResult: TestRunResult) {
-        val runIdentifier = ReportCoordinates(
-            planSlug = "AvitoAndroid",
-            jobSlug = "FunctionalTests",
-            runId = "49.0.275.32855"
-        )
-        val reportViewer = StubReportViewer(reportViewerUrl)
+//        val runIdentifier = ReportCoordinates(
+//            planSlug = "AvitoAndroid",
+//            jobSlug = "FunctionalTests",
+//            runId = "49.0.275.32855"
+//        )
+//        val reportViewer = StubReportViewer(reportViewerUrl)
```

Comment:
Do we need the commented code?

Reply:
can't see it but not changed it either 😕

Reply:
I've reviewed by commits. Maybe it was an intermediate state.